stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelController.kt

```diff
@@ -85,13 +83,6 @@ public sealed interface ChannelController {
     public val recoveryNeeded: Boolean
     public val cid: String
 
-    @Deprecated(
-        level = DeprecationLevel.ERROR,
-        message = "Use offlineChannelData instead of it",
-        replaceWith = ReplaceWith("ChannelController::offlineChannelData"),
-    )
-    public val channelData: LiveData<ChannelData>
-
     public val offlineChannelData: LiveData<OfflineChannelData>
```

Comment:
And rename it to `channelData`

Reply:
This will be a breaking change. Do you think we should do it without deprecation flow?

Reply:
Yes, you're right. Let's leave it as is